pkg/controller/acmeorders/sync.go

```diff
@@ -285,6 +290,11 @@ func (c *controller) updateOrderStatus(ctx context.Context, cl acmecl.Interface,
 	if err != nil {
 		return nil, err
 	}
+
+	return c.updateOrderStatusFromACMEOrder(ctx, cl, o, acmeOrder)
+}
+
+func (c *controller) updateOrderStatusFromACMEOrder(ctx context.Context, cl acmecl.Interface, o *cmacme.Order, acmeOrder *acmeapi.Order) (*acmeapi.Order, error) {
```

Comment:
^ Git displays the diff in a weird way- I split out part of the `updateOrderStatus` function into a new `updateOrderStatusFromACMEOrder` func that can be called with an existing ACME Order